### ISSUE TITLE
Site as an effective advert for the community

### ISSUE THREAD

**Issue by cassgvp** [cassgvp/WIN-Open-Neuroimaging-Community#1]
After having a quick look through the [Open WIN Community](https://cassgvp.github.io/WIN-Open-Neuroimaging-Community/) pages:

- [ ] Does the website make you want to join the community?
- [ ] Do you want to know more? 
- [ ] Is there any specific information which is missing or hidden?
- [ ] Was the website easy to use?

Please feed back on any aspect which you do or don't enjoy or understand. This could be navigation, structure, colours, content, whatever you think would attract you or importantly might cause others to turn away.

**Comment by alonbaram2** [comment]
Hi!

I thought the website was very well built and clear. The information about the community, how to join and future plans are comprehensive and it's very accessible. It made me join the Slack channel, that I didn't know existed 👍 
My only comment is about the tools section. While the tools are very well explained, it takes time to understand that most of them don't yet exist / there is no documentation yet of how to use them. I think that can be a bit misleading for people who are looking for these kinds of solutions now. For example, in the "open data" section the XNAT server is described as if it's already up and running. Only once you get to the "How to use" section do you realise that this is a future plan and doesn't yet exist. This is true of other tools too. I think it might be useful to put a more noticeable note somewhere so that people don't get frustrated by trying to understand how to use tools that are not yet ready to use.  

But overall - this website definitely makes me more likely to try to contribute :)

**Comment by cassgvp** [comment]
Comment from Verena Heise:

- I love it, it’s amazing, well done you!!!

# General comment
- I hope you can use some of the educational materials that I and others wrote for the original WIN Open neuroimaging website. I remember I wrote a guide on open publishing for example. I am personally a big fan of structuring the resources we develop around the stages of a research project so that people can check what they should be thinking about at the current stage of their project, but we might also want to do this differently, totally up to you.

**Comment by cassgvp** [comment]
> My only comment is about the tools section. While the tools are very well explained, it takes time to understand that most of them don't yet exist / there is no documentation yet of how to use them. ...

Thank you for pointing this out @alonbaram2 . Totally valid point! I've updated each of the tool descriptions and the main tools page to have a "coming soon" note at the top (for example [here on the main tools page](https://cassgvp.github.io/WIN-Open-Neuroimaging-Community/docs/tools.html)). 

Would you be able to have a look and see this this works for you?

**Comment by jessica-walsh** [comment]
Hi Cass! I think the website looks brilliant and is very open about all future plans. I love the fact you’ve put the roadmap with timescales up there for everyone to see. It's really clear about what you are aiming for and when people can expect to see developments! 
I am wondering whether it would be useful to have another section on something like ‘tips for being an open researcher’ which would include smaller scale things that individual researchers can do on a daily basis - such as sharing early stage ideas with colleagues, seeking feedback, looking out for collaborations, share pre-prints of papers, publishing in open access journals etc? It’s just an idea, but all of the open WIN tools are larger-scale things, whereas it might also be nice to have smaller things that people can be trying to adopt into their research now? 

A couple of general technical things:
- On the roadmap page - Milestone 6 draft program link doesn’t work
- On the open research calendar page it says ‘List of 2013 Events’

A couple of more general ideas:
- This may be completely outside the open WIN remit - but what about people who want do not want to share their data/tools openly right now but are looking for collaboration? e.g. a clinician with a rare dataset who is looking for someone with the right tool to help analyse it, or an image analyst with a new tool who is looking for an interesting dataset to apply them to. I’m wondering if there could be a (v easy to use) way of putting a kind of ‘advert’ up to announce that you have this data/tools and for anyone interested to contact you? 
- Using the ambassador idea, I’m wondering if you could expand the ambassadors (once up and running) to include people external to WIN  who use the WIN open data/tools? It would help foster external collaborations and grow the community

**Comment by TulikaN** [comment]
Great work from the entire team! I'm really looking forward to seeing the tools up and running.

Couple minor things I noticed -
1. The open Canvas images are a bit difficult to read. Is it possible to make the sidebar smaller and increase the text size in the images?
2. A few typos in - Personas> What do they need to be motivated to engage and sustain their engagement?; Join us at a documentation hack! > 2nd paragraph says 'shearing' instead of sharing.

**Comment by ludogriffanti** [comment]
Great website Cass and others!
I also joined the slack straight away (although I did not find the #welcome channel mentioned - does it exist?)
I also look forward to the documentation hack!

I think the quick links to the main areas are great, while I found the "Open Leadership" section as the first heading under Home a bit difficult to understand without looking at the rest of the website first. I was not expecting to find that type of information at the very beginning. If I understood well it is the framework that this project is based on, but the more practical aims are under "Using this repository" and "Contributing". Might be worth revisiting the order of the sections and/or call the "Open Leadership" section something like "Principles guiding this project".

A couple of minor edits/typos under "Open Analysis":
- FSL this is the tool > FSL is the tool
- The Working Group has devised a processes  > a process
- Unclear sentence overall "The Working Group has devised a processes which researchers can slot into their normal FSL usage to ensure their analysis is reproducible and can be easily shared with others.", although it might get clearer when more content is added.

I like Verena's suggestion of having a summary page with the steps of a typical research project with links to how to make each step more open. Most of them will send you to the tools already available on the website, plus I can see info on publishing, like the open access info that was presented a while ago at WIN.

**Comment by MalikaIhle** [comment]
great work Cass and team! addition of coming soon message is great, I also found the open canvas picture hard to read, and also found the open leadership and goals of the program under open ambassador hard to grasp on their own. it almost feel like internal information (e.g. Demonstrate infrastructure engagement, Open Canvas) for which you want to be open about but I think following the suggestion of ludogriffanti would be a good way to make sure this information isn't the first thing people get to.

**Comment by cassgvp** [comment]
> 1. The open Canvas images are a bit difficult to read. Is it possible to make the sidebar smaller and increase the text size in the images?

Also received feedback that the blue font on white background isn't great for accessibility, so I'll look to redraw this set. 

> 2. A few typos in - Personas> What do they need to be motivated to engage and sustain their engagement?; Join us at a documentation hack! > 2nd paragraph says 'shearing' instead of sharing.

Thank you for pointing out typos! Im sure there are many!! I'm going to create a "good first issue" of correcting typos :)

**Comment by falfaroalmagro** [comment]
Hi Cass,

Love the site! I will open a new issue as you suggested by email.

**Comment by falfaroalmagro** [comment]
Short comment: The target for the site is not very clear to me. Is the site intended for people at WIN or people out of WIN? Are we showing what we are doing in terms of open-science to outer people (i.e. self-promotion)? Is the site intended for people out of WIN to give them access to our open-science materials (tools/datasets)? Is the site intended to WIN people to give them the information they may need (guides/ideas)? Should it have a proselytising angle or should we presume that people visiting know the basics about open science and are in favour?

I think that the target audience determines the content of the site, and I am not very sure it is possible to create a site for all these different audiences :)

**Comment by mchiew** [comment]
Just a comment after attempting to find and join the Slack. I think a "Join the OpenWIN Slack" link or button on the main home page might be useful. My first instinct was to click on the "Community" heading, but joining the Slack didn't seem to be in any of those pages. I clicked around a bit, but I had trouble finding it even knowing it existed somewhere. I finally resorted to the Search function, which did work, but isn't super discoverable for those not already looking for a Slack.

**Comment by jvohryzek** [comment]
Hello Cass
Lots of great work and material on the website:) Well done!
I have very much enjoyed sharing the milestones list as a feature.

I am sending some feedback below; feel free to use it or discard it.

I have got two main points regarding the layout

1) Upon reading the content, I found the structure very straightforward to follow. However,  I got slightly lost with the different dynamic links which lead to either a new page or reference a content further down the page. Ans so I was wondering whether there is a way to simplify the navigation to the different sections. The same applies to the two ways of navigating the content; one on the left side of the page and one as part of the main pages text.

2) I also like Verena's comment on structuring some of the content as one would use it when getting on with they research project. It might be a fairly intuitive way of processing the various information about open-science.

some further points:

1) I thought the calendar with the future events is a great idea and it made me wonder whether it might be worth to also include a news tab with other open-science events. It might give sense of broader community in which this initiative is happening. To name a few events; the Alan Turing Institute Collaborative Hackathons or the Brainhack Global.( I guess it would also require an active engagement from at least one ambassador.)

2) Would stating the mission of the community clearly on the main page be a good idea?

3) Perhaps, I might have missed it; would it make sense to motivate the benefits of open-science with some relevant literature sources?

some minor points:

1) perhaps it has ben mentioned previously; some of the figure captions are difficult to read ie the graphs in 'Goals of the programme' section or the 'Who are we' section

2) the 'who are we' section has a 'Who is the community' as a main title.

3) I will follow up with a few typos I spotted in the relevant issue:)

Again, great job! and please feel free to use whatever comments you deem relevant.

**Comment by cassgvp** [comment]
> Short comment: The target for the site is not very clear to me. Is the site intended for people at WIN or people out of WIN? Are we showing what we are doing in terms of open-science to outer people (i.e. self-promotion)? Is the site intended for people out of WIN to give them access to our open-science materials (tools/datasets)? Is the site intended to WIN people to give them the information they may need (guides/ideas)? Should it have a proselytising angle or should we presume that people visiting know the basics about open science and are in favour?
> 
> I think that the target audience determines the content of the site, and I am not very sure it is possible to create a site for all these different audiences :)

Thank you for this feedback @falfaroalmagro. At the moment I have mostly been focussing on an internal audience, but with a view to it also being visible to external users to understand what we are doing. There would likely be more information for external users about how to access our shared research outputs from the main WIN website. But I totally get your comment and we'll need to think carefully about who this site is serving and make it explicit. 

> should we presume that people visiting know the basics about open science and are in favour?

Feedback from others has suggested that we shouldn't assume this, and there is a new issue #6 around creating (or curating) this material. I feel this is separate from the main focus of the site which is around how to use the new tools, but again will need to think carefully about how to position/structure it.

**Comment by cassgvp** [comment]
> Just a comment after attempting to find and join the Slack. I think a "Join the OpenWIN Slack" link or button on the main home page might be useful. My first instinct was to click on the "Community" heading, but joining the Slack didn't seem to be in any of those pages. I clicked around a bit, but I had trouble finding it even knowing it existed somewhere. I finally resorted to the Search function, which did work, but isn't super discoverable for those not already looking for a Slack.

Thank you @mchiew. There is a "Join the community" link at the top of each page, but seems this didn't catch you. Feedback from others has been that having slack as the only advertised means of communication could be a bit of a barrier, so perhaps this button/link could be "Join us on Slack or by email"?  I've suggested this in a new issue #9 . Could you let me know if you think this might do the job?

**Comment by cassgvp** [comment]
> Hello Cass
> Lots of great work and material on the website:) Well done!
> I have very much enjoyed sharing the milestones list as a feature.
> 
> I am sending some feedback below; feel free to use it or discard it.
> 
> I have got two main points regarding the layout
> 
> 1. Upon reading the content, I found the structure very straightforward to follow. However,  I got slightly lost with the different dynamic links which lead to either a new page or reference a content further down the page. Ans so I was wondering whether there is a way to simplify the navigation to the different sections. The same applies to the two ways of navigating the content; one on the left side of the page and one as part of the main pages text.
> 2. I also like Verena's comment on structuring some of the content as one would use it when getting on with they research project. It might be a fairly intuitive way of processing the various information about open-science.
> 
> some further points:
> 
> 1. I thought the calendar with the future events is a great idea and it made me wonder whether it might be worth to also include a news tab with other open-science events. It might give sense of broader community in which this initiative is happening. To name a few events; the Alan Turing Institute Collaborative Hackathons or the Brainhack Global.( I guess it would also require an active engagement from at least one ambassador.)
> 2. Would stating the mission of the community clearly on the main page be a good idea?
> 3. Perhaps, I might have missed it; would it make sense to motivate the benefits of open-science with some relevant literature sources?
> 
> some minor points:
> 
> 1. perhaps it has ben mentioned previously; some of the figure captions are difficult to read ie the graphs in 'Goals of the programme' section or the 'Who are we' section
> 2. the 'who are we' section has a 'Who is the community' as a main title.
> 3. I will follow up with a few typos I spotted in the relevant issue:)
> 
> Again, great job! and please feel free to use whatever comments you deem relevant.

These are great @jvohryzek , thank you. I think we should probably break these down into individual issues. Some of these are issues with the specific theme ([just-the-docs](https://pmarsceill.github.io/just-the-docs/) and my limited hacking of the navigation structure. We're ultimately going to port this whole site over to the WIN GitLab instance, so the whole theme might be rejigged. If you're around for the [first hack day](https://cassgvp.github.io/WIN-Open-Neuroimaging-Community/docs/events/doc-hack-1.html), we could play with some of this then?

**Comment by jvohryzek** [comment]
Wonderful. As you suggested @cassgvp , I have created two potential issues.

**Comment by jvohryzek** [comment]
> > Hello Cass
> > Lots of great work and material on the website:) Well done!
> > I have very much enjoyed sharing the milestones list as a feature.
> > I am sending some feedback below; feel free to use it or discard it.
> > I have got two main points regarding the layout
> > 
> > 1. Upon reading the content, I found the structure very straightforward to follow. However,  I got slightly lost with the different dynamic links which lead to either a new page or reference a content further down the page. Ans so I was wondering whether there is a way to simplify the navigation to the different sections. The same applies to the two ways of navigating the content; one on the left side of the page and one as part of the main pages text.
> > 2. I also like Verena's comment on structuring some of the content as one would use it when getting on with they research project. It might be a fairly intuitive way of processing the various information about open-science.
> > 
> > some further points:
> > 
> > 1. I thought the calendar with the future events is a great idea and it made me wonder whether it might be worth to also include a news tab with other open-science events. It might give sense of broader community in which this initiative is happening. To name a few events; the Alan Turing Institute Collaborative Hackathons or the Brainhack Global.( I guess it would also require an active engagement from at least one ambassador.)
> > 2. Would stating the mission of the community clearly on the main page be a good idea?
> > 3. Perhaps, I might have missed it; would it make sense to motivate the benefits of open-science with some relevant literature sources?
> > 
> > some minor points:
> > 
> > 1. perhaps it has ben mentioned previously; some of the figure captions are difficult to read ie the graphs in 'Goals of the programme' section or the 'Who are we' section
> > 2. the 'who are we' section has a 'Who is the community' as a main title.
> > 3. I will follow up with a few typos I spotted in the relevant issue:)
> > 
> > Again, great job! and please feel free to use whatever comments you deem relevant.
> 
> These are great @jvohryzek , thank you. I think we should probably break these down into individual issues. Some of these are issues with the specific theme ([just-the-docs](https://pmarsceill.github.io/just-the-docs/) and my limited hacking of the navigation structure. We're ultimately going to port this whole site over to the WIN GitLab instance, so the whole theme might be rejigged. If you're around for the [first hack day](https://cassgvp.github.io/WIN-Open-Neuroimaging-Community/docs/events/doc-hack-1.html), we could play with some of this then?

Wonderful. As you suggested @cassgvp , I have created two potential issues.

**Comment by cassgvp** [comment]
Thanks for these, @jessica-walsh 

> A couple of general technical things:
> 
> * On the roadmap page - Milestone 6 draft program link doesn’t work

I've fixed this link (and updated completed tasks) in ae3757734938d5dc9d1a3a8221d8475ac9eb7ca7

> * On the open research calendar page it says ‘List of 2013 Events’

Not quite sure what is causing this! The whole section is an embedded google calendar, but I'm not sure why it thinks it's starting from "2013". I'll look into it in issue #13 .

> A couple of more general ideas:
> 
> * This may be completely outside the open WIN remit - but what about people who want do not want to share their data/tools openly right now but are looking for collaboration? e.g. a clinician with a rare dataset who is looking for someone with the right tool to help analyse it, or an image analyst with a new tool who is looking for an interesting dataset to apply them to. I’m wondering if there could be a (v easy to use) way of putting a kind of ‘advert’ up to announce that you have this data/tools and for anyone interested to contact you?

This is an interesting idea. I think ultimately once the shared data are externally available this would be easier to achieve. We would of course want our data to be index appropriately so it could be found by things like [Google Dataset Search](https://datasetsearch.research.google.com), but that relies on someone actively looking for your data. This idea has come up in other places (something like a "jobs board"), but I think it would be better positioned as an international effort rather than something WIN specific. 

Hot off the press: A very similar idea proposed by Simon Eichoff (a Clinician by training): https://twitter.com/INM7_ISN/status/1304405331485708290?s=20

> * Using the ambassador idea, I’m wondering if you could expand the ambassadors (once up and running) to include people external to WIN  who use the WIN open data/tools? It would help foster external collaborations and grow the community

That would be very cool 😎

**Comment by cassgvp** [comment]
### General note on typos! 

If you uncover any, please could you add them via a comment on issue #7 ? I'll then collate and add them to the issue description.

Thank you! 🙏🏻

**Comment by carobellum** [comment]
Hi Cass,

great job on the website! As for your questions: 
The website is really engaging, structure easy to follow and slick design - all in all, makes me want to browse and engage with the community.  Usability is great, the search function works well and the categories make sense.

From a practical perspective, there might be a couple of things that I would have found useful. When explaining de-identification steps such as defacing, it would be great to have an example of what that looks like. I'm thinking of Fidel's amazing paper on the UK Biobank processing pipeline that has example images to show what defacing looks like (Alfaro-Almagro2018). For someone unfamiliar with de-identification steps this would help them understand why this is important and show what this looks like. MIght even be useful to provide two views of the defaced image, one with the face reconstructed to show how the facial features are removed and one with a sagittal slice to show that this doesn't affect the brain tissue.

It would also be great if there was an example of a risk assessment on the website so that researchers who want to share their data get an idea of how to set something like this up. I couldn't find a mention of the risk assessment on the website, so maybe I haven't found the right section. But I remember when I started out, writing the risk assessment was the part I could find the least information about. 

All in all, amazing job! Very excited to see this go up!

**Comment by cassgvp** [comment]
Thanks @carobellum , all great suggestions. Lots of content we can pull in on the [hack day](https://cassgvp.github.io/WIN-Open-Neuroimaging-Community/docs/events/doc-hack-1.html)

**Comment by JasleenKJolly** [comment]
You have done a brilliant job. It is great to see it all come together. The website was easy to follow. It was great to be able to able to navigate from the menu on the side or the embedded links. The flow was logical and exciting. I have some comments about readability but will raise an issue
